public/_headers

```diff
@@ -1,14 +1,12 @@
 /*
-    Cache-Control: public, max-age=0, s-maxage=31556926, must-revalidate
-
-/
+    Cache-Control: public, max-age=0, must-revalidate
     Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
```

Comment:
there is a small change in here too, in that the `Strict-Transport-Security` header is now under `/*`, vs `/` before.

The header is used for [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security), and is injected into every response by Netlify anyhow.
However, right now `/` responds with `Strict-Transport-Security: max-age=63072000; includeSubDomains; preload` (as seen here), whereas every other page responds with the Netlify default of `strict-transport-security: max-age=31536000` - which is a bit inconsistent...